pkg/app/context.go

```diff
@@ -1045,6 +1045,20 @@ func (ctx *RequestContext) Cookie(key string) []byte {
 	return ctx.Request.Header.Cookie(key)
 }
 
+// Cookies returns all the request cookies.
+//
+// It's a good idea to call protocol.ReleaseCookie to reduce GC load after the cookie used.
+func (ctx *RequestContext) Cookies() []*protocol.Cookie {
+	var cookies []*protocol.Cookie
+	ctx.Request.Header.VisitAllCookie(func(key, value []byte) {
+		cookie := protocol.AcquireCookie()
+		cookie.SetKeyBytes(key)
+		cookie.SetValueBytes(value)
```

Comment:
why not use `cookie.Parse`

Reply:
Because the value here is the value of each cookie, different from the value when you call `VisitAllCookie` through `ctx.Response.Header`, which is the value of `Set-Cookie` field. It's simpler to use `Set` directly.

Reply:
cc @li-jin-gou